<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteBookWidget</class>
 <widget class="AbstractFileEditor" name="NoteBookWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NoteBookWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,2">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="noteLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="fmtEditLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="NoSinkingToolButton" name="fmtBrushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">padding-left:0px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../notebook_resources.qrc">
           <normaloff>:/assets/icon/formatbrush.svg</normaloff>:/assets/icon/formatbrush.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="fontComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::WheelFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fontSizeComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>6</number>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertPolicy::NoInsert</enum>
         </property>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>28</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>36</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>48</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>72</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="ColorPicker" name="colorPickerWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NoSinkingToolButton" name="boldButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">padding-left:0px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../notebook_resources.qrc">
           <normaloff>:/assets/icon/bold.svg</normaloff>:/assets/icon/bold.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NoSinkingToolButton" name="italicButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">padding-left:0px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../notebook_resources.qrc">
           <normaloff>:/assets/icon/italic.svg</normaloff>:/assets/icon/italic.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NoSinkingToolButton" name="underlineButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">padding-left:0px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../notebook_resources.qrc">
           <normaloff>:/assets/icon/underline.svg</normaloff>:/assets/icon/underline.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NoSinkingToolButton" name="strikeOutButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">padding-left:0px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../notebook_resources.qrc">
           <normaloff>:/assets/icon/strikeout.svg</normaloff>:/assets/icon/strikeout.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="MultiStatePushButton" name="calqlatrToggleButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="AdvTextEdit" name="mainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CalqlatrWidget" name="calqlatrWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalqlatrWidget</class>
   <extends>QWidget</extends>
   <header>CalqlatrWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AdvTextEdit</class>
   <extends>QTextEdit</extends>
   <header>AdvTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>AbstractFileEditor</class>
   <extends>QWidget</extends>
   <header>AbstractFileEditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NoSinkingToolButton</class>
   <extends>QToolButton</extends>
   <header>NoSinkingToolButton.h</header>
  </customwidget>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QWidget</extends>
   <header>ColorPicker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MultiStatePushButton</class>
   <extends>QPushButton</extends>
   <header>MultiStatePushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../notebook_resources.qrc"/>
 </resources>
 <connections/>
</ui>
